core/workspace_svg.js

```diff
@@ -991,6 +1004,36 @@ Blockly.WorkspaceSvg.prototype.paste = function(xmlBlock) {
   block.select();
 };
 
+/**
+ * Paste the provided block onto the workspace.
+ * @param {!Element} xmlComment XML workspace comment element.
+ */
+Blockly.WorkspaceSvg.prototype.pasteWorkspaceComment_ = function(xmlComment) {
+  Blockly.Events.disable();
+  try {
+    var comment = Blockly.WorkspaceCommentSvg.fromXml(xmlComment, this);
+    // Move the duplicate to original position.
+    var commentX = parseInt(xmlComment.getAttribute('x'), 10);
+    var commentY = parseInt(xmlComment.getAttribute('y'), 10);
+    if (!isNaN(commentX) && !isNaN(commentY)) {
+      if (this.RTL) {
+        commentX = -commentX;
+      }
+      // Offset workspace comment.
+      // TODO: properly offset comment such that it's not interfereing with any blocks
```

Comment:
Make an issue for this and add a reference.  I'll add it to the project on github.  That way we don't lose track of it.